tests/test_custom_type_setup.cpp

```diff
@@ -13,14 +13,19 @@
 
 namespace py = pybind11;
 
+PYBIND11_NAMESPACE_BEGIN(PYBIND11_NS_VISIBILITY(pybind11_tests))
```

Comment:
Not sure if I understand need for applying `hidden` when it only contains an anonymous namespace.

Is it possible to revert this (possibly redundant) change?

Reply:
Thanks Eric, I'll wait for Henry's opinion before investing more time, but to explain quickly, this is purely to appease some sad compilers that don't "realize" that the unnamed namespace is already hidden. I could add comments, or remove the unnamed namespace, I'll figure out something after we decided the big question.